dex/networks/dcr/script.go

```diff
@@ -784,41 +784,53 @@ func ExtractContractHash(scriptHex string) ([]byte, error) {
 // contract must be provided for the search algorithm to verify the correct data
 // push. Only contracts of length SwapContractSize that can be validated by
 // ExtractSwapDetails are recognized.
-func FindKeyPush(sigScript, contractHash []byte, chainParams *chaincfg.Params) ([]byte, error) {
-	dataPushes, err := txscript.PushedData(sigScript)
```

Comment:
For reference when we look back at this diff: https://github.com/decred/dcrd/commit/06cd8b0b377f9ba3b614be2bb970e25f95d85fa7